include/proxy-wasm/exports.h

```diff
@@ -148,9 +149,35 @@ Word wasi_unstable_sched_yield();
 Word wasi_unstable_poll_oneoff(Word in, Word out, Word nsubscriptions, Word nevents);
 void wasi_unstable_proc_exit(Word);
 Word wasi_unstable_clock_time_get(Word, uint64_t, Word);
+Word wasi_unstable_clock_res_get(Word, Word);
+Word wasi_unstable_fd_advise(Word, uint64_t, uint64_t, Word);
+Word wasi_unstable_fd_allocate(Word, uint64_t, uint64_t);
+Word wasi_unstable_fd_datasync(Word);
+Word wasi_unstable_fd_fdstat_set_rights(Word, uint64_t, uint64_t);
+Word wasi_unstable_fd_filestat_set_size(Word, uint64_t);
+Word wasi_unstable_fd_filestat_set_times(Word, uint64_t, uint64_t, Word);
+Word wasi_unstable_fd_pread(Word, Word, Word, uint64_t, Word);
+Word wasi_unstable_fd_pwrite(Word, Word, Word, uint64_t, Word);
+Word wasi_unstable_fd_readdir(Word, Word, Word, uint64_t, Word);
+Word wasi_unstable_fd_renumber(Word, Word);
+Word wasi_unstable_fd_sync(Word);
+Word wasi_unstable_fd_tell(Word, Word);
+Word wasi_unstable_path_create_directory(Word, Word, Word);
+Word wasi_unstable_path_filestat_set_times(Word, Word, Word, Word, uint64_t, uint64_t, Word);
+Word wasi_unstable_path_link(Word, Word, Word, Word, Word, Word);
+Word wasi_unstable_path_readlink(Word, Word, Word, Word, Word, Word);
+Word wasi_unstable_path_remove_directory(Word, Word, Word);
+Word wasi_unstable_path_rename(Word, Word, Word, Word, Word);
+Word wasi_unstable_path_symlink(Word, Word, Word, Word);
+Word wasi_unstable_path_unlink_file(Word, Word, Word);
+Word wasi_unstable_sock_accept(Word, Word, Word);
+Word wasi_unstable_sock_recv(Word, Word, Word, Word, Word, Word);
+Word wasi_unstable_sock_send(Word, Word, Word, Word, Word);
+Word wasi_unstable_sock_shutdown(Word, Word);
```

Comment:
Presumably, those socket functions are not "randomly" imported by the Go runtime, and if we add stubs for them, then we're replacing load time failure with a runtime failure.

There is a difference between existing functions (bare minimum required by runtimes to load targets compiled to a WASI architecture) vs adding everything without implementing it.